src/components/Results.svelte

```diff
@@ -8,12 +8,13 @@
   let focusedIdx;
 
   // Reset focused result to be first result when the results get updated
+  let flag = false;
   const unsubscribe = results.subscribe(() => {
     focusedIdx = keyNavArray($results);
+    flag = $results.length>1?true:false;
```

Comment:
A separate variable for this small change is not necessary here. We can move this ternary conditional to the template itself.

Reply:
Okay Sure. Will update that.

Reply:
I updated the corresponding file. Have a look.